src/Api/Controllers/OrganizationsController.cs

```diff
@@ -410,10 +410,10 @@ public async Task Delete(string id, [FromBody]OrganizationDeleteRequestModel mod
                 throw new UnauthorizedAccessException();
             }
 
-            if (!await _userService.CheckPasswordAsync(user, model.MasterPasswordHash))
+            if (!await _userService.VerifyPasswordOrOTPAsync(user, model.Secret))
             {
                 await Task.Delay(2000);
-                throw new BadRequestException("MasterPasswordHash", "Invalid password.");
+                throw new BadRequestException(string.Empty, "User verification failed.");
```

Comment:
Why string.Empty for the key here instead of 'Secret'?

Reply:
We usually use the explicit field, since it could be either `MasterPasswordHash` or `OTP` we omit it and give a generic error.